CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ## 1.35.0 - (September 22, 2020)
 
+* Changed
```

Comment:
Can you move this into the Unreleased section? This probably just got merged weird with main.

Reply:
Updated here https://github.com/cerner/terra-application/pull/89/commits/3b6dae1c224f32327c8058d8bcc8b7280c108efa